scripts/stop_connection.py

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to add a general connection to the database.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from hera_mc import mc, cm_utils, part_connect, cm_handling
+import sys
+import copy
+
+
+def query_args(args):
+    """
+    Gets information from user
+    """
+    if args.uppart is None:
+        args.uppart = raw_input("Upstream part number:  ")
+    if args.uprev is None:
+        args.uprev = raw_input("Upstream part revision:  ")
+    if args.upport is None:
+        args.upport = raw_input("Upstream output port:  ")
+    if args.dnpart is None:
+        args.dnpart = raw_input("Downstream part number:  ")
+    if args.dnrev is None:
+        args.dnrev = raw_input("Downstream part revision:  ")
+    if args.dnport is None:
+        args.dnport = raw_input("Downstream input port:  ")
+    if args.date == 'now':
+        args.date = cm_utils._query_default('date', args)
+    return args
+
+
+if __name__ == '__main__':
```

Comment:
It can be convenient to group all of the `__main__` functionality in a single function so that the bottom of the script ends up looking like this:

```
if __name__ == '__main__':
    main()
```